samples/ClassroomLabs/Scripts/HyperV/SetupForNestedVirtualization.ps1

```diff
@@ -0,0 +1,288 @@
+<#

```

Comment:
The README.md is recognized as a binary file. Please fix that.

Since an update is required, can you also please ensure to remove all Tab characters and that the indentation and formatting is correct?

Reply:
Both documents have been reformatted.